0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -123,6 +124,11 @@ Unsubscribe from ledger events.
 
 Query whether the plugin is currently connected.
 
+#### isOpen
+<code>ledgerPlugin.isOpen() ⇒ Boolean</code>
+
+Query whether the plugin is currently connected and has an active websocket.
```

Comment:
Isn't this what `isConnected` is for? If plugin bells isn't currently giving useful information when that it called, the implementation of that method should be changed rather than adding a new one to the interface.